interface/forms/painmap/FormPainMap.php

```diff
@@ -26,14 +26,21 @@ class FormPainMap extends AbstractClickmapModel
      *
      * @var TABLE_NAME
      */
+
     static $TABLE_NAME = "form_painmap";
 
-    /* Initializer. just alles parent's initializer. */
-    function FormPainMap($id = "")
+    /* Initializer. just calls parent's initializer. */
+    function __construct($id = "")
     {
+        parent::__construct();
         parent::AbstractClickmapModel(FormPainMap::$TABLE_NAME, $id);
     }
 
+    function __toString()
+    {
+        return "ID: " . $this->id . "\n";
+    }
```

Comment:
If there is no toString in a class, then I wouldn't add it. Since a toString should be generating a string of all the variables(ie. no reason to add a function that is not correct).